README.md

```diff
@@ -154,41 +243,32 @@ npm start
 ```
 
 
-## 5. RIF Storage Pinning service
+## 7. RIF Storage Pinning service
 
 > This is a service that listens on blockchain events and when new Agreement is created it pins a file to the configured IPFS node.
 
-Download and setup the Pinning service
+Download and setup the Pinning service (already done when running **IPFS** nodes)
 ```
-$ git clone git@github.com:rsksmart/rif-storage-ipfs-pinner.git
-$ cd rif-storage-ipfs-pinner
+$ git clone git@github.com:rsksmart/rif-storage-pinner.git
+$ cd rif-storage-pinner
 $ npm i
 ```
 
-Install IPFS. Recommended way is using [ipfs-update](https://github.com/ipfs/ipfs-update) but there are other ways described [here](https://github.com/ipfs/go-ipfs#install) as well. Recommended is to run the latest version but required is at least `0.5.0`.
-
-Initialize development repos that are placed in `.repos`.  This folder can be anytime removed and the `init` command rerun. All data will be purged though.
-
-```
-$ npm run init
-```
+Make sure you have **IPFS** installed. We will use one of the previously deployed instances of **IPFS**.
 
-Spawn IPFS daemons
+To interact with pinning service use the `npm run bin` script. To start Pinning service run:
 
 ```
-$ npm run ipfs:consumer daemon
-$ npm run ipfs:provider daemon
+npm run bin -- init --offerId=0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1 --db=./db.sqlite
 ```
 
-You can use NPM's scripts `npm run ipfs:consumer` and `npm run ipfs:provider` to interact with each IPFS nodes. It has the same commands like `ipfs` command.
-
-To interact with pinning service use the `npm run bin` script. To start Pinning service run:
+This will provide the `peerId` that should be used in the *RIF Marketplace UI*  to create the Storage offer. Once the offer is created in the UI you can run the service using:
```

Comment:
```suggestion
This will provide the `peerId` that should be used in the *RIF Marketplace UI*  to create the Storage offer. You can create that offer on `http://localhost:3000/storage/sell?peerId={your_peer_id}`.
Once the offer is created in the UI you can run the service using:
```